src/test/java/com/wikia/webdriver/testcases/mercurytests/old/LoginTestsOld.java

```diff
@@ -129,7 +129,7 @@ public void MercuryLoginTest_011_japaneseUserLogIn() {
 //        Configuration.getCredentials().userNameJapanese2));
   }
 
-  @Test(groups = "MercuryLoginTest_012")
+  @Test(groups = {"MercuryLoginTest_012", "LoginTests_002"})
   public void MercuryLoginTest_012_passwordTogglerWorks() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)